src/WheelsAndGoods.Api/Configuration/Startup.cs

```diff
@@ -74,6 +81,11 @@ public void ConfigureServices(IServiceCollection services)
 
 				options.OperationFilter<AuthOperationFilter>();
 			});
+		
+		services.AddScoped<IUnitOfWork, UnitOfWork>();
```

Comment:
Lets move to https://github.com/kirpichyov/WheelsAndGoods_BE/blob/main/src/WheelsAndGoods.Application/ServiceCollectionExtensions.cs